include/psa/crypto.h

```diff
@@ -39,6 +39,18 @@ extern "C" {
  * algorithms, key types, policies, etc. */
 #include "crypto_types.h"
 
+/* The file "crypto_sizes.h" contains definitions for size calculation
+ * macros whose definitions are implementation-specific. */
+#include "crypto_sizes.h"
```

Comment:
`crypto_sizes.h` uses macros from `crypto_values.h`, so it would make more sense to include `crypto_values.h` first. I don't think it actually matters for the sake of the C language, because `crypto_sizes.h` itself only defines macros, it doesn't expand them. But then `crypto_struct.h` does expand macros from `crypto_sizes.h`, and that can result in expanding macros from `crypto_values.h`. It works because `crypto_struct.h` (indirectly) includes `crypto_driver_common.h` which includes `crypto_values.h`. But it would make more sense to include `crypto_values.h` at the logical place in `crypto.h`.

Reply:
Right, I missed that, thanks. I've changed it and while at it I've tried to fix the similar issues in crypto_extra.h pointed to by #7087.